internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -305,20 +296,25 @@ func (r *BundleReconciler) handleCompletedPod(ctx context.Context, u *updater.Up
 }
 
 func (r *BundleReconciler) getBundleContents(ctx context.Context, pod *corev1.Pod) (fs.FS, error) {
-	bundleContentsData, err := r.getPodLogs(ctx, pod)
+	url := fmt.Sprintf("http://%s:8080/content", pod.Status.PodIP)
```

Comment:
Is this a stable IP that we can rely on, or should we have a service in front of the unpacker pod?

Reply:
Almost positive the pod IP is fixed for the lifetime of the pod. Even if it isn't we read the pod status on every reconcile to get the IP, so I think we're good either way?